lib/nerves_runtime/log/parser.ex

```diff
@@ -1,28 +1,49 @@
 defmodule Nerves.Runtime.Log.Parser do
   @moduledoc """
-  Functions for parsing syslog (RFC 5424) strings
+  Functions for parsing syslog and kmsg strings
   """
 
   @doc """
   Parse out the syslog facility, severity, and message (including the timestamp
   and host) from a syslog-formatted string.
+
+  The message is of the form:
+
+  <pri>message
+
+  where pri is an integer that when broken apart gives you a facility and severity.
+  message is everything else.
```

Comment:
Last sentence ends in a period, this sentence is a fragment and needs capitalization.